x-pack/plugins/cases/common/types/domain/connector/v1.ts

```diff
@@ -118,6 +119,21 @@ const ConnectorSwimlaneTypeFieldsRt = rt.strict({
   fields: rt.union([SwimlaneFieldsRt, rt.null]),
 });
 
+/**
+ * Thehive
+ */
+
+export const TheHiveFieldsRt = rt.strict({
+  tlp: rt.union([rt.number, rt.string, rt.null]),
```

Comment:
The schema of the `tlp` of the connector is `number` and not a string. I think we should keep the same convention here for cases (`number` or `null`). This way we can avoid doing `parseInt(tlp, 10)` on the backend. We may need to do it on the frontend as the `select` dropdown accepts only strings as values (as you do here `x-pack/plugins/stack_connectors/public/connector_types/thehive/params_case.tsx`).